### PR TITLE
Migrate setup.py to setup.cfg

### PR DESCRIPTION
**Pull request by Uxio0**
- Remove pyproject.toml, it was breaking local install (`pip -e ...`)

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Gnosis Ltd
+Copyright (c) 2018 Safe Ecosystem Foundation
```
[the remark this answers is not in the document]
Comment (Uxio0) [reply LICENSE:3]: Well, it was suposed to be updated, so I just fixed it. If you think it's necessary I will create a new PR 😉

**Review by moisses89 — APPROVED** [review]
Looks good for me! One thing, I understand that this is going to increase the version to 4.5.0.

File: LICENSE (hunk 1 above)
[the remark this answers is not in the document]
Comment (fmrsabino) [reply LICENSE:3]: I also agree with @moisses89 but just because I think it brings more context to the changes going in.

File: setup.cfg [hunk 2]
```diff
@@ -1,3 +1,68 @@
+[metadata]
+name = safe-eth-py
+version = 4.5.0
```
Comment (fmrsabino) [inline setup.cfg:3]: The version changed here. Not sure if this is intended.

Comment (Uxio0) [reply setup.cfg:3]: Yes, it's intended, I wanted to have the version ready asap so I can use it on the tx service and deploy it next week

**Pull request merged**